admin_manual/configuration_server/occ_command.rst

```diff
@@ -276,13 +294,61 @@ This is example converts to SQLite MySQL/MariaDB::
 For a more detailed explanation see 
 :doc:`../configuration_database/db_conversion`
 
+.. _encryption_label:
+
 Encryption
 ----------
 
-When you are using encryption, you must manually migrate your encryption 
-keys after upgrading your ownCloud server::
+ownCloud 8.2 introduces a new set of encryption commands::
+
+ encryption:change-key-storage-root  
+ encryption:show-key-storage-root    
+ encryption:set-default-module       
+ encryption:list-modules             
+ encryption:encrypt-all              
+ encryption:decrypt-all              
+ encryption:disable                  
+ encryption:status                   
+ encryption:enable
+ encryption:enable-master-key 
+ encryption:migrate
+
+``encryption:status`` shows whether you have active encryption, and your default encryption module. To enable encryption you must first enable the Encryption app, and then run ``encryption:enable``::
+
+ $ sudo -u www-data php occ app:enable encryption
+ $ sudo -u www-data php occ encryption:enable
+ $ sudo -u www-data php occ encryption:status
+  - enabled: true
+  - defaultModule: OC_DEFAULT_MODULE
+   
+``encryption:change-key-storage-root`` is for moving your encryption keys to a different folder. It takes one argument, ``newRoot``, which defines your new root folder::
+
+ $ sudo -u www-data php occ encryption:change-key-storage-root /etc/oc-keys
+ 
+You can see the current location of your keys folder::
+
+ sudo -u www-data php occ encryption:show-key-storage-root
+ Current key storage root:  default storage location (data/)
+ 
+``encryption:list-modules`` displays your available encryption modules. You will see a list of modules only if you have enabled the Encryption app. Use ``encryption:set-default-module [module name]`` to set your desired module.
+
+``encryption:encrypt-all`` encrypts all data files for all users. You must first put your ownCloud server into single-user mode to prevent any user activity until encryption is completed.
```

Comment:
the script will enable single-user mode automatically, so no need to do it manually before the admin run the script